src/components/LocaleSelector.vue

```diff
@@ -0,0 +1,79 @@
+<!--
+  This source file is part of the Swift.org open source project
+
+  Copyright (c) 2023 Apple Inc. and the Swift project authors
+  Licensed under Apache License v2.0 with Runtime Library Exception
+
+  See https://swift.org/LICENSE.txt for license information
+  See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+-->
+
+<template>
+  <div class="locale-selector">
+    <select
+      v-model="$i18n.locale"
+      :aria-label="$t('select-language')"
+    >
+      <option
+        v-for="(lang, i) in langs"
+        :key="`lang-${i}`"
+        :value="lang"
+      >
+        {{ getLanguageName(lang) }}
+      </option>
+    </select>
+    <ChevronThickIcon class="icon-inline" />
+  </div>
+</template>
+
+<script>
+import ChevronThickIcon from 'theme/components/Icons/ChevronThickIcon.vue';
+import locales from 'docc-render/lang/locales.json';
+
+export default {
+  name: 'LocaleSelector',
+  components: {
+    ChevronThickIcon,
+  },
+  data() {
+    return {
+      langs: Object.keys(locales),
```

Comment:
Is this variable ever expected to change within the lifetime of this component? If not, I think a constant and/or computed property would make more sense than having this be a `data` item.

Reply:
same https://github.com/marinaaisa/swift-docc-render/pull/8/